### PR TITLE
feat: extract config for finngen ingestion to yaml file

### PR DESCRIPTION
**Pull request by project-defiant**
# Context

Move all configuration of the finngen_ingestion dag to the separate config.
This dag will has it's own lifecycle depending on the [finngen data releases](https://finngen.gitbook.io/documentation), so it's a good idea to have it run on demand only when requred and cache the produced results:
- finngen credible sets (ingested by gentropy StudyLocus class)
- finngen study index (ingested by gentropy StudyIndex class)
inside the staging gcs path `gs://finngen_data/` with appropriate release prefix (`r11`)

# Implementations
- [x] extracted constants from the `finngen_ingestion.py` to `finngen_ingestion.yaml` config file.
- [x] Explicitly declared parameters that were used as default in gentropy finngen steps.
- [x] Renamed dag

### CODE REVIEW

File: src/ot_orchestration/dags/finngen_ingesion.py
Comment (javfg) [inline src/ot_orchestration/dags/finngen_ingesion.py:1]: There is a typo in the filename. `finngen_ingesion.py` -> `finngen_ingestion.py`.

File: src/ot_orchestration/dags/finngen_ingesion.py [hunk 1]
```diff
@@ -15,18 +15,10 @@
     install_dependencies,
     submit_step,
 )
+from ot_orchestration.utils.utils import read_yaml_config
```
Comment (javfg) [inline src/ot_orchestration/dags/finngen_ingesion.py]:
```suggestion
from ot_orchestration.utils import read_yaml_config
```

You export `read_yaml_config` in the `__init__` file.

**Review by javfg — CHANGES_REQUESTED** [review]

File: src/ot_orchestration/dags/finngen_ingesion.py (hunk 1 above)
Comment (project-defiant) [reply src/ot_orchestration/dags/finngen_ingesion.py]: you are right, I am not too consistent about my `__all__` flags :p. This was IDE that pointed be out to the wrong subpackage.

**Comment by project-defiant** [comment]
@javfg ready for 🥊  2

**Review by javfg — APPROVED** [review]

**Comment by javfg** [comment]
All good this time around!

**Pull request merged**